week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -4,13 +4,24 @@
   Create a function which:
   - Accepts an array which contains all the students' names and their attendance counts
     (see tests to confirm how this data will be structured)
-  - Returns an array containing onl the names of the who have attended AT LEAST 8 classes
+  - Returns an array containing only the names of the students who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(arr) {
+  
+  let students = arr.filter(x => x[1] >= 8).map(x => x.splice(0, 1).join(" "))
```

Comment:
Excellent work! I love the *chaining* here! 💪 

There's 1 little thing that you could improve. If you look at exercise 3, you'll see there's a different (and I think better) way of getting the first letter from a string. Could we use that here instead of the `.splice()` and `.join()`?

Reply:
it would be more concise if I would have x[0] instead of using splice and join